python/global_data_process.py

```diff
@@ -26,7 +26,7 @@
 timelist = np.array(timelist.split(",")).astype(str)
 
 """User Options"""
-path = "/groups/ESS/whung/canopy_wind/gfsv16_test_data"  # work directory
+path = "./input"  # work directory
```

Comment:
@angehung5 But running "python global_data_process.py" in python directory as in README suggests,  then this implies ../input right?
```suggestion
path = "../input"  # work directory
```

Reply:
Or, we need to change the README to be same as running slurm script as:  "python python/global_data_process.py"

Reply:
It should be fine. Just all relative directories should be relative to where the slurm script is. Users can always change them if needed.